dsl/docker-multiarch/images/python.groovy

```diff
@@ -34,10 +34,12 @@ for v in */; do
 	docker build -t "$repo:$v" "$v"
 	docker build -t "$repo:$v-onbuild" "$v/onbuild"
 	docker build -t "$repo:$v-slim" "$v/slim"
+	docker build -t "$repo:$v-alpine" "$v/alpine"
```

Comment:
I see what you mean. I'm totally cool with you taking over. You're much more knowledgeable on how this works.